pkg/proxy/healthcheck/healthcheck.go

```diff
@@ -335,13 +342,20 @@ func (h healthzHandler) ServeHTTP(resp http.ResponseWriter, req *http.Request) {
 	if val := h.hs.lastUpdated.Load(); val != nil {
 		lastUpdated = val.(time.Time)
 	}
+	apiLastUpdated := time.Time{}
+	if val := h.hs.apiLastUpdated.Load(); val != nil {
+		apiLastUpdated = val.(time.Time)
```

Comment:
It would be better to handle type assertion error.

Reply:
Ack think I can do that better